data/resources/style.css

```diff
@@ -148,6 +148,45 @@ sidebarsearchitemrow {
   color: @accent_fg_color;
 }
 
+messagedocument > box {
+  padding: 9px;
+  border-radius: 12px;
```

Comment:
```suggestion
```
IMHO, this looks much better in combination with hover effect. To make this work, you have to set `<property name="overflow">hidden</property>` in `content-message-document.ui`.

![grafik](https://user-images.githubusercontent.com/3630213/189540934-c123655a-3a71-4445-9078-f222f5af40a6.png)

![grafik](https://user-images.githubusercontent.com/3630213/189541028-d63faba5-3a61-4761-b84f-9090ab57637f.png)

I think that it would even look better if the sender label had a 3px margin at the bottom